functions-and-scope.js

```diff
@@ -71,6 +111,45 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 9
+let highestGrade;
+function checkHighestGrade(grades){
+    highestGrade= 0;
+    if (grades.includes(10)){
+        highestGrade = 10;
+    }
+    else if (grades.includes(9)){
+        highestGrade = 9;
+    }
+    else if (grades.includes(8)){
+        highestGrade = 8;
+    }
+    else if (grades.includes(7)){
+        highestGrade = 7;
+    }
+    else if (grades.includes(6)){
+        highestGrade = 6;
+    }
+    else if (grades.includes(5)){
+        highestGrade = 5;
+    }
+    else if (grades.includes(4)){
+        highestGrade = 4;
+    }
+    else if (grades.includes(3)){
+        highestGrade = 3;
+    }
+   else if (grades.includes(2)){
+        highestGrade = 2;
+    }
+    else if (grades.includes(1)){
+        highestGrade = 1;
+    }
+    return highestGrade;
```

Comment:
Dit is niet echt schaalbaar. Ik zal je laten zien hoe het simpeler kan:
```suggestion
    let highestGrade = grades[0]; // belangrijk om de eerste waarde te pakken in plaats van 0. Anders werkt te functie niet met de array [-4, -2, -8, -3]
    for (let grade of grades){
        if (grade > highestGrade){
            highestGrade = grade;
        }
    }
    
    return highestGrade;
```